<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPlainTextEdit" name="historyMsg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>710</width>
     <height>160</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBtn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>590</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testBtn">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>590</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>模拟违法</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>350</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>350</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>350</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>350</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ipLabel0">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string>255.255.255.255</string>
   </property>
  </widget>
  <widget class="QLabel" name="ipLabel1">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>190</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string>255.255.255.255</string>
   </property>
  </widget>
  <widget class="QLabel" name="ipLabel2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string>255.255.255.255</string>
   </property>
  </widget>
  <widget class="QLabel" name="ipLabel3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>400</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string>255.255.255.255</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
